src/components/leaflet/WisMap.vue

```diff
@@ -153,8 +153,7 @@ export default defineComponent({
         })
         .catch(function () {
           self.$root.catch(`
-            <p>${self.$t("messages.does_not_exist")}</p>
-            <p>${self.$t("messages.how_to_link_station")}</p>`);
+            <p>${self.$t("messages.does_not_exist")}</p>`);
```

Comment:
this change would means that any issue with the association with stations results in the "does not exist" message ?